### PR TITLE
[Docs] Consolidate community install methods

### PR DESCRIPTION
**Pull request by DerMolly**
### Component/Part
Docs

### Description
This PR adds all community install methods to a single page. This seems to be a better option as all of these methods have very little to document and this makes it easier to browse through the different methods.

### Steps

- [x] Added / updated documentation
- [x] I read the [contribution documentation](https://github.com/hedgedoc/hedgedoc/blob/develop/CONTRIBUTING.md) and
  made sure that:
  - My commits are signed-off to accept the DCO.
  - This PR targets the correct branch: `master` for 1.x & docs, `develop` for 2.x

### CODE REVIEW

File: docs/content/dev/getting-started.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## Preparing for running the code
 
-**Notice:** *There's [specialised instructions for docker](../setup/docker.md) or [heroku](../setup/heroku.md), if you prefer running code this way!*
+**Notice:** *There's [specialised instructions for docker](../setup/docker.md) if you prefer running code this way!*
```
Comment (davidmehren) [inline docs/content/dev/getting-started.md:5]:
```suggestion
**Notice:** *There are [specialised instructions for docker](../setup/docker.md) if you prefer running code this way!*
```

File: docs/content/setup/community.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
+# Community installation methods
```
Comment (davidmehren) [inline docs/content/setup/community.md:1]:
```suggestion
# Community Installation Methods
```

File: docs/content/setup/community.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+The HedgeDoc community has created and tested many options for deploying HedgeDoc on other platforms or operating systems.
+You can check them out below. Feel free to create a PR to add your tested community installation method.
+
+## Container Deployments
+
+### LinuxServer.io Docker image
```
Comment (davidmehren) [inline docs/content/setup/community.md:8]:
```suggestion
### LinuxServer.io Docker Image
```

File: docs/content/setup/community.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
+# Community installation methods
+
+The HedgeDoc community has created and tested many options for deploying HedgeDoc on other platforms or operating systems.
```
Comment (davidmehren) [inline docs/content/setup/community.md:3]:
```suggestion
The HedgeDoc community has created and tested many options for deploying HedgeDoc on other platforms or operating systems.
These are not officially supported by the HegeDoc maintainers. If you encounter issues or have questions, please first reach out to downstream packagers.
```

File: docs/content/setup/community.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+You can check them out below. Feel free to create a PR to add your tested community installation method.
+
+## Container Deployments
+
+### LinuxServer.io Docker image
+
+[![Discord](https://img.shields.io/discord/354974912613449730.svg?color=94398d&labelColor=555555&logoColor=ffffff&style=for-the-badge&label=Discord&logo=discord)](https://discord.gg/YWrKVTn "realtime support / chat with the community and the team.")
+[![GitHub Release](https://img.shields.io/github/release/linuxserver/docker-hedgedoc.svg?color=94398d&labelColor=555555&logoColor=ffffff&style=for-the-badge&logo=github)](https://github.com/linuxserver/docker-hedgedoc/releases)
+
+[LinuxServer.io](https://linuxserver.io) have created an Ubuntu-based multi-arch container image for x86-64, arm64 and armhf.
```
Comment (davidmehren) [inline docs/content/setup/community.md:13]:
```suggestion
[LinuxServer.io](https://linuxserver.io) have created an Alpine-based multi-arch container image for x86-64, arm64 and armhf.
```
https://github.com/linuxserver/docker-hedgedoc/blob/main/Dockerfile#L1

File: docs/mkdocs.yml [hunk 6]
```diff
@@ -12,14 +12,7 @@ nav:
...
           - 'Manual Installation': setup/manual-setup.md
-          - Heroku: setup/heroku.md
-      - Community installation methods:
-          - 'LinuxServer Docker Image': setup/docker-linuxserver.md
-          - 'Helm Chart': setup/kubernetes-helm.md
-          - 'YunoHost App': setup/yunohost.md
-          - 'Cloudron App': setup/cloudron.md
-          - 'Arch Linux Package': setup/arch-linux.md
-          - 'PikaPods Hosting': setup/pikapods.md
+      - 'Community installation methods': setup/community.md
```
Comment (davidmehren) [inline docs/mkdocs.yml:15]:
```suggestion
      - 'Community Installation Methods': setup/community.md
```

**Review by davidmehren — CHANGES_REQUESTED** [review]

File: docs/content/setup/community.md (hunk 2 above)
Comment (ErikMichelson) [reply docs/content/setup/community.md]: fixed

File: docs/content/setup/community.md (hunk 3 above)
Comment (ErikMichelson) [reply docs/content/setup/community.md]: fixed

File: docs/mkdocs.yml (hunk 6 above)
Comment (ErikMichelson) [reply docs/mkdocs.yml]: fixed

**Review by davidmehren — APPROVED** [review]

**Pull request merged**